cjio/geom_help.py

```diff
@@ -39,6 +39,8 @@ def get_normal_newell(poly):
         ne = i + 1
         if (ne == len(poly)):
             ne = 0
+        if len(poly[i]) < 3 or len(poly[ne]) < 3:
```

Comment:
Actually not 100% sure with it being here and checking for poly elements. I had it outside before with `if len(poly) < 3`, but that also felt weird. lmk what would be best